src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private BufferedReader bufferedReader;
+    private String[] arrayToWriteIntoFile = new String[3];
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+        try {
+            bufferedReader = new BufferedReader(new FileReader(fromFileName));
+            String value = bufferedReader.readLine();
```

Comment:
split all code from this method to the smaller ones, and just call them from this method
![image](https://github.com/mate-academy/jv-work-with-file/assets/73041091/03a19791-49bf-4d75-98ad-8991e9f904a9)